qtconsole/tests/test_completion_widget.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import unittest
+
+from qtconsole.qt import QtCore, QtGui
+
+from qtconsole.qt_loaders import (loaded_api, QT_API_PYQT, QT_API_PYQT5,
+QT_API_PYQTv1, QT_API_PYQT_DEFAULT, QT_API_PYSIDE)
+
+from qtconsole.console_widget import ConsoleWidget
+from qtconsole.completion_widget import CompletionWidget
+import ipython_genutils.testing.decorators as dec
+
+api = loaded_api()
+if api in (QT_API_PYQT, QT_API_PYQTv1, QT_API_PYQT_DEFAULT):
+    from PyQt4.QtTest import QTest
+elif api == QT_API_PYQT5:
+    from PyQt5.QtTest import QTest
+elif api == QT_API_PYSIDE:
+    from PySide.QtTest import QTest
```

Comment:
I think this could be simplified by only importing `QT_API_PYQT5` and `QT_API_PYSIDE` and writing this block as

```python
if api == QT_API_PYQT5:
    from PyQt5.QtTest import QTest
elif api == QT_API_PYSIDE:
    from PySide.QtTest import QTest
else:
    from PyQt4.QtTest import QTest
```